sci-biology/discrover/discrover-1.5.2.ebuild

```diff
@@ -37,16 +37,28 @@ DEPEND="${RDEPEND}
 		dev-texlive/texlive-latexrecommended
 		media-gfx/imagemagick
 	)
+	lto? ( >=sys-devel/gcc-4.8 )
 "
+pkg_pretend() {
+	if use lto; then
+		if [[ $(gcc-major-version) -lt 4 ]] || ( [[ $(gcc-major-version) -eq 4 && $(gcc-minor-version) -lt 8 ]] ) ; then
+			eerror "Compilation with link-time optimization and GCC older than 4.8 is not supported."
+			eerror "Please either disable the USE flag 'lto' or use >=sys-devel/gcc-4.8."
+			die "Compiling with USE flag 'lto' is not supported with <sys-devel/gcc-4.8."
+		fi
+	fi
+}
 src_configure() {
 	local mycmakeargs=(
 		$(cmake-utils_use_with cairo CAIRO)
 		$(cmake-utils_use_with dreme DREME)
 		$(cmake-utils_use_with doc DOC)
+		$(cmake-utils_use_with lto LTO)
 		$(cmake-utils_use_with tikzlogo TIKZLOGO)
 		$(cmake-utils_use_with misc_scripts MISC_SCRIPTS)
 		$(cmake-utils_use_with rmathlib RMATHLIB)
 		$(cmake-utils_use_with tcmalloc TCMALLOC)
+		-DDOC_DIR=/usr/share/doc/${PF}
```

Comment:
Yeah, after consulting the documentation, that seems like a good idea. I might do it, or something similar.
I'm still checking however, if it's not possible to use something like ${CMAKE_INSTALL_PREFIX}/share/doc/${PF} this would be the most appropriate, I think. I have to figure out if I can use that variable (it's certainly passed to cmake so I should be able to figure it out.) Pointers welcome! :)